framework/SupervisedLearning/ARMA.py

```diff
@@ -42,6 +42,18 @@
 from .SupervisedLearning import supervisedLearning
 #Internal Modules End--------------------------------------------------------------------------------
 
+def _toStr(s):
+  """
+    Removes unicode from strings in Python 2 so amsc can use it.
+    @ In, s, unicode or str, String to convert to plain str
+    @ Out, s, str, Converted str
+  """
+  if sys.version_info.major > 2:
+    return s
+  return s.encode('ascii')
```

Comment:
as above

Reply:
+ the docstrings is actually copied/pasted

Reply:
Removed.